### DISCUSSION TITLE
Add multiselection

### DISCUSSION THREAD

**Discussion by vitonsky** [community/community#8598] [Other Feature Feedback, Questions, & Ideas]
Please, implement selection of items from favorites to batch manipulations over stars.

When i see my stars i want to select multiple repos and then add/delete it all to some collections

It will make github a great tool to creating repository collections

**Comment by brianlovin** [comment]
Agreed, thanks for the suggestion @vitonsky — I've added your +1 to our team's tracking issue for this.

**Comment by adamcstephens** [comment]
I have 890 stars, which I'd love to sort into lists. Doing so would pretty much require bulk editing of some sort.

**Comment by 192933488S** [comment]
This is clearly NEEDED. After so many years everybody has dozens of stars or repos to sort out.

**Comment by liaoxuewei** [comment]
I support this suggestion very much. Lists are very practical. Gitee already has them:)
